src/component/selection/getSelectionOffsetKeyForNode.js

```diff
@@ -18,7 +18,8 @@
  * found on the DOM tree of given node.
  */
 function getSelectionOffsetKeyForNode(node: Node): ?string {
-  if (node instanceof Element) {
+  var win = node.ownerDocument.defaultView;
+  if (win && node instanceof win.Element) {
```

Comment:
node.nodeType === 1

Reply:
As above, this needed `typeof node.getAttribute === 'function'`

Reply:
Sorry, this comment is incorrect. Here, it’s solved by casting `node` as an `Element`, as in `var element: Element = (node: any);`